### PR TITLE
docs: fix comment type

### LINKED ISSUE #77: Comment type is func, should be comment
Documentation Is:

- [ ] Missing
- [ ] Needed
- [x] Confusing
- [ ] Not Sure?

### Please Explain in Detail...

Type of Comment is documented as `func`.
https://github.com/shellscape/postcss-values-parser/blob/master/docs/Comment.md#type

### Your Proposal for Changes

Should be `comment`.

### LINKED ISSUE #79: Exports second interpolation should be variables
Documentation Is:

- [ ] Missing
- [ ] Needed
- [x] Confusing
- [ ] Not Sure?

### Please Explain in Detail...

Second `interpolation` should be `variables`.
https://github.com/shellscape/postcss-values-parser/blob/master/docs/Exports.md#interpolation-1

### Your Proposal for Changes

rename to `variables`

### PR DESCRIPTION
**Pull request by AndyOGo**
This PR fixes #77 and fixes #79 and contains:

- [x] bugfix
- [ ] feature
- [ ] refactor
- [ ] tests
- [ ] documentation
- [ ] metadata

### Breaking Changes?

- [ ] yes
- [x] no

If yes, please describe the breakage.

### CODE REVIEW

File: docs/Exports.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Default: `false`
 Set this option to enable parsing of interpolated values for languages such as SCSS. For example:
 `interpolation: { prefix: '@' }` will allow parsing of the interpolated value `@{batman}` which uses `@` as the "prefix". For SCSS one might use `interpolation: { prefix: '#' }`.
 
-##### `interpolation`
+##### `variables`
```
Comment (AndyOGo) [inline docs/Exports.md]: @shellscape 
I guess `parentNode` is missing 🤔 
https://github.com/shellscape/postcss-values-parser/blob/ed608973a1fcda642f48a3126695b8c837dfacf1/lib/ValuesParser.js#L24-L32

Comment (shellscape) [reply docs/Exports.md]: `parentNode` shouldn't be explicitly set by the user, so it was left out on purpose :)

**Comment by shellscape** [comment]
Cheers, thank you! 🍺

**Comment by AndyOGo** [comment]
Glad to help, just minor things... Thank you for this awesome lib:)

**Pull request merged**